pkg/cli/admin/catalog/build.go

```diff
@@ -44,25 +67,69 @@ type BuildImageOptions struct {
 	FilterOptions   imagemanifest.FilterOptions
 	ParallelOptions imagemanifest.ParallelOptions
 
-	FromFileDir string
-	FileDir     string
+	FromFileDir  string
+	FileDir      string
+	ReleaseImage string
 }
 
-const defaultFromImage = "quay.io/openshift/origin-operator-registry:latest"
-
 func NewBuildImageOptions(streams genericclioptions.IOStreams) *BuildImageOptions {
-	// Adjust default build opts to use an OpenShift base "from" image
-	defaults := appregistry.DefaultAppregistryBuildOptions()
-	defaults.From = defaultFromImage
-
+	// From must be specified, or we must infer it from a release
+	o := appregistry.DefaultAppregistryBuildOptions()
+	o.From = ""
 	return &BuildImageOptions{
-		AppregistryBuildOptions: defaults,
+		AppregistryBuildOptions: o,
 		IOStreams:               streams,
 		ParallelOptions:         imagemanifest.ParallelOptions{MaxPerRegistry: 4},
 	}
 }
 
-func (o *BuildImageOptions) Complete(cmd *cobra.Command, args []string) error {
+func (o *BuildImageOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []string) error {
+	// expect to extract the base image from an active ocp connection or a release image
+	if len(o.From) == 0 {
+		fmt.Fprintf(o.IOStreams.Out, "--from not specified, attempting to get base image from release image\n")
```

Comment:
These look more like debugging messages, rather than normal output. If you want to, I'd put one statement at the end where you get the image from.